packages/fiori-elements-writer/src/data/defaults.ts

```diff
@@ -88,5 +88,11 @@ export function setAppDefaults<T>(feApp: FioriElementsApp<T>): FioriElementsApp<
             feApp.ui5.version ?? TemplateTypeAttributes[feApp.template.type].minimumUi5Version[feApp.service.version]!;
     }
 
+    // if not explictely disabled, enable the SAP Fiori tools
+    feApp.appOptions = feApp.appOptions ?? {};
+    if (feApp.appOptions.sapux !== false) {
+        feApp.appOptions.sapux = true;
+    }
```

Comment:
Need to consider if this would impact CAP where FE does not have sapux = true.

Reply:
I would consider it an exceptional case, so the Fiori generator would have to set `sapux: false` explicitly for CAP.

Reply:
The flag indicates that the generated project is supported by the Fiori tools. We want people to be able to use the tools with things we know we can support, so the flag should always have been true. Nevertheless, if the caller explicitly sets it to `false` it is respected.
So for the CAP case, where we need it differently, I expect that when this is integrated by the Fiori generator, that someone now sets it explicitly to false. Current version of the fe write is `0.4.7`, this one changes it to `0.5.0` so it is aligned with semver (minor change below 1)